### DISCUSSION TITLE
Spawn new window with same tab/pane

### DISCUSSION THREAD

**Discussion by psydvl** [wezterm/wezterm#2745] [Q&A]
I'd want to create shortcut to spawn a new window with current pane
Other words: I start to build something in pane and want to see this pane overflowing. 
How can I attach old pane to new window?
Or maybe reattach?

**Comment by yutkat** [comment]
If you want to use wezterm like tmux, you could use the multiplexing feature.
https://wezfurlong.org/wezterm/multiplexing.html

**Reply by psydvl** [reply]
So, you recommend spawning each new tab in new unix_domain?

**Reply by yutkat** [reply]
I'm not sure because I've never wanted to do what you're saying. Why do you want to open the same tab/pane configuration in different windows?

If you want to use the same tab/pane configuration on a different PC, the multiplexing feature is enough.

**Reply by psydvl** [reply]
Not configuration, tab/pane itself, like [wezterm cli move-pane-to-new-tab](https://wezfurlong.org/wezterm/cli/cli/move-pane-to-new-tab.html#wezterm-cli-move-pane-to-new-tab) do

**Reply by yutkat** [reply]
Oh, I see what you mean. You just wanted to make Pane to Tab.

**Comment by psydvl** [comment]
I've found [wezterm cli move-pane-to-new-tab](https://wezfurlong.org/wezterm/cli/cli/move-pane-to-new-tab.html#wezterm-cli-move-pane-to-new-tab), but none related lua methods
